src/login/controller/login.ts

```diff
@@ -73,8 +73,20 @@ class LoginController extends Controller {
       return;
     }
 
-    ctx.response.headers.set('Location', '/');
+    if (this.isRedirect()) {
+      const redirectVal = getSetting('login.defaultRedirect');
+      ctx.response.headers.set('Location', redirectVal);
+    } else {
+      ctx.response.headers.set('Location', '/');
+    }
+
+  }
 
+  isRedirect() {
+    if (getSetting('login.defaultRedirect') !== '/') {
+      return true;
+    }
+    return false;
```

Comment:
```suggestion
    ctx.response.redirect(303, getSetting('login.defaultRedirect', '/'));
```

Reply:
I think this will work

Reply:
ahhhh yeah, i guess function has default params :( my bad